src/global/propagation.rs

```diff
@@ -21,39 +21,39 @@ lazy_static::lazy_static! {
 /// let propagator = api::TraceContextPropagator::new();
 ///
 /// // assign it as the global propagator
-/// global::set_http_text_propagator(propagator);
+/// global::set_text_map_propagator(propagator);
 /// ```
-pub fn set_http_text_propagator<P: api::HttpTextFormat + Send + Sync + 'static>(propagator: P) {
-    let _lock = GLOBAL_HTTP_TEXT_PROPAGATOR
+pub fn set_text_map_propagator<P: api::TextMapFormat + Send + Sync + 'static>(propagator: P) {
+    let _lock = GLOBAL_TEXT_MAP_PROPAGATOR
         .write()
         .map(|mut global_propagator| *global_propagator = Box::new(propagator));
 }
 
-/// Executes a closure with a reference to the current global [`HttpTextFormat`] propagator.
+/// Executes a closure with a reference to the current global [`TextMapFormat`] propagator.
 ///
-/// [`HttpTextFormat`]: ../api/context/propagation/trait.HttpTextFormat.html
+/// [`TextMapFormat`]: ../api/context/propagation/trait.TextMapFormat.html
 ///
 /// # Examples
 ///
 /// ```
-/// use opentelemetry::{api, api::HttpTextFormat, global};
+/// use opentelemetry::{api, api::TextMapFormat, global};
 /// use std::collections::HashMap;
 ///
 /// let example_carrier = HashMap::new();
 ///
 /// // create your http text propagator
 /// let tc_propagator = api::TraceContextPropagator::new();
-/// global::set_http_text_propagator(tc_propagator);
+/// global::set_text_map_propagator(tc_propagator);
 ///
 /// // use the global http text propagator to extract contexts
```

Comment:
So close, sorry missed these the first time around, maybe find and replace for `http text`? 😄

Reply:
Ugh, can't believe I missed those. Sorry about that. Fixed! And this time I am pretty confident that we got them all 🤞